### PR TITLE
Gourmonger Tweaks again

### PR DESCRIPTION
**Pull request by Jackreturnswithaveangenace**
Lowering the health and speed so people can deal with them a bit easier.

Edit: As per suggestions I've exchanged some of the nerf on the health for making them unable to smash through *reads code* literally every single obstacle possible including adminbus walls (except shuttles).

:cl:
 * tweak: Gourmongers are now actually fat aka slow. Also soft and squishy since they weren't meant for the battlefield. They can also no longer smash through ultra-triple reinforced adminwalls, roid walls, etc on their way to their target.

### CODE REVIEW

**Comment by Eneocho** [comment]
Unless you make gours less LOOSE-y, as in they can't break doors or go through walls or whatever, you won't really make gours viable, they're too easily LOOSE'd. Rubylips' idea was an easy to setup, hard to contain engine, but all the designs currently discovered for it require tons of work, to the point you'd be better off making a SME.

And the main problem with gours as an antag weapon isn't their speed or health, it's the ungodly amounts of radiation that sends almost everyone to toxin-slow/vomit-stun in 1 second of exposure which enables the gour to waddle to the body and eat it. They're basically a walking active shard that doesn't shit plasma.

**Comment by Jackreturnswithaveangenace** [comment]
One solution would involve actually fixing the code to make them viable, another would involve removing them. I've opted just to make them easier to kill in this PR, but I'm not against commenting them out or making them much harder to get since at the moment they're just an antag device.

**Comment by Optimism333** [comment]
> Unless you make gours less LOOSE-y, as in they can't break doors or go through walls or whatever, you won't really make gours viable, they're too easily LOOSE'd. And the main problem with gours as an antag weapon isn't their speed or health, it's the ungodly amounts of radiation that sends almost everyone to toxin-slow/vomit-stun in 1 second of exposure which enables the gour to waddle to the body and eat it.

Basically agree with what Eneocho said. They're not actually THAT tanky in my opinion. The health isn't the problem, it's the rads, speed, and the fact they can Kool-Aid smash their way through any walls in their path.

I'd suggest leaving the health as is (or maybe just reducing it to 100), keeping the speed reduction, and looking at ways to reduce the ease with which they can escape containment.

**Comment by Jackreturnswithaveangenace** [comment]
I've removed its smash containers flag to see if that helps with the smashing through stuff (it should help with some at least) and what I _think_ is the proc for how close you need to be as a living mob before you get blasted with radiation, down from almost screen wide to being in 2 tiles range.

**Comment by Eneocho** [comment]
Keep in mind that reducing the radiation area also affects how useful they are as an engine, keep it to 3 tiles, ideally (since that's the radius of a standard shard room).

**Comment by Optimism333** [comment]
Perhaps ask Rubylips if there's a way to change this, if it's still true:

Screenshot (413)

There should be SOME way to guarantee that they'll remain contained even if they're starving, barring someone loosing them on purpose.

**Comment by Eneocho** [comment]
> 
> There should be SOME way to guarantee that they'll remain contained, barring outside intervention.

The only design I know works is making an area with teleporters so the gours are trapped inside and if they loose, they get teleported to the beacon inside again. Which goes against the easy to start hard to contain idea of the gours, since it's a massive endeavour to make the teleporters.

**Comment by dunsel-hordon** [comment]
What is wrong with you people??? Making a gour proof cage is simple. Just setup teleporters and have the beacon in the center of em. Stop being dumb. Also its easy and fast to make. Just use the mechanics flatpack machines.

**Comment by Eneocho** [comment]
> 
> 
> What is wrong with you people??? Making a gour proof cage is simple. Just setup teleporters and have the beacon in the center of em. Stop being dumb. Also its easy and fast to make. Just use the mechanics flatpack machines.

I literally just said that. 

And using the mechanics flatpacks requires mining, materials and cooperation from the assistant+, so again, goes against the idea of gours being simple to start.

**Comment by dunsel-hordon** [comment]
> > What is wrong with you people??? Making a gour proof cage is simple. Just setup teleporters and have the beacon in the center of em. Stop being dumb. Also its easy and fast to make. Just use the mechanics flatpack machines.
> 
> I literally just said that.
> 
> And using the mechanics flatpacks requires mining, materials and cooperation from the assistant+, so again, goes against the idea of gours being simple to start.

na its literally the easiest shit ever and you can have it ready to go in 10 - 15 minutes. Doesn't get much simpler than that.

**Comment by Eneocho** [comment]
> na its literally the easiest shit ever and you can have it ready to go in 10 - 15 minutes. Doesn't get much simpler than that.

Lets say you're right, it still doesn't change that gours are incredibly unwieldy, requiring that specific setup to use, without any kind of customization. The fun with most engines is how autistic can you get with the design, tweaking the piping, structure, plasma, etc, but gour straight up demands the telecage or for them to be built offstation or you fucking LOOSE.

File: code/modules/power/gourmonger.dm [hunk 1]
```diff
@@ -12,16 +12,15 @@ var/global/gourmonger_saturation = 0
...
+	maxHealth = 60
 	melee_damage_lower = 5
 	melee_damage_upper = 10	//The rads are the real danger
 	stat_attack = DEAD	//So it attacks corpses.
 	search_objects = 1	//Searches objects but doesn't ignore people. The ignoring before hanger is in its CanAttack()
-	environment_smash_flags = SMASH_CONTAINERS
 	wanted_objects = list(/obj/item/weapon/reagent_containers/food/snacks)
 	meat_amount = 1
-	speed = 1.2	//Able to just barely outrun them
+	speed = 2	//Able to just barely outrun them -- How are these fat fucks able to move so fast?
```
Comment (Eneocho) [inline code/modules/power/gourmonger.dm:23]: They aren't fat fucks, that's the point, they're literally a stomach with legs that instantly turns food into rads.

Comment (Jackreturnswithaveangenace) [reply code/modules/power/gourmonger.dm:23]: Sounds fat to me

Comment (Eneocho) [reply code/modules/power/gourmonger.dm:23]: you can't be fat if you don't produce fat

**Comment by Durgen** [comment]
> > > What is wrong with you people??? Making a gour proof cage is simple. Just setup teleporters and have the beacon in the center of em. Stop being dumb. Also its easy and fast to make. Just use the mechanics flatpack machines.
> > 
> > 
> > I literally just said that.
> > And using the mechanics flatpacks requires mining, materials and cooperation from the assistant+, so again, goes against the idea of gours being simple to start.
> 
> na its literally the easiest shit ever and you can have it ready to go in 10 - 15 minutes. Doesn't get much simpler than that.

are you memeing, it takes way longer than that. First you need to wait for miners to get mats (or mine them yourself) , then scan, print, and transport 28 flatpacks (for a 3x3 cage). Then you have to move rad collectors, (and place wires for them), as well as come up with some method to spam the gours with infinite food. (usually requiring even more flatpacks)

All this effort, and you are on thin ice. The first time there is a power outage, or a clown comes in and turns off a telegenerator, everything goes tits up.

image

**Comment by KillFurries** [comment]
Stop nerfing the gourmonger and just give up on the stupid vore engine that never works and no one ever tries anymore.
You've created a mob that is completely useless, it can't be used for engines and only retards will try to use it for engines, it's now too weak to loose and will just die. Then what's the point of the mob?
The vore engine idea sucks. Give up.

**Comment by Jackreturnswithaveangenace** [comment]
You should be able to use objects to pen it in now at least instead of it being a snowflaked mob that penetrates even the hardest metal known to man.

**Comment by Eneocho** [comment]
> 
> 
> You should be able to use objects to pen it in now at least instead of it being a snowflaked mob that penetrates even the hardest metal known to man.

Now that you fixed the main problem (it being able to cuck literally any containment barring teleportation or it being in space) it wouldn't be a bad idea to restore their speed and health, to make them still useable as an antag weapon/have some risk attached to the engine.

**Comment by Optimism333** [comment]
On some further thought, I think this pr should be more atomic, with the intent of solving or toning down one or two of the more annoying things about them (ex: the large radiation radius and/or the ease of containing them). A 50% reduction of their health and big nerf to speed on top seems too much.

If you really want tweaks made, I think it would also be worthwhile to give some feedback directly to Rubylips. He might be willing to make some changes himself.

**Comment by dunsel-hordon** [comment]
you only need 5 gold, silver, uranium, and plasma. Then before you are mining order additional metal glass and plastic. That all will take you at most 10 minutes to do. The longest part is literally waiting on the flatpack machine to print you everything. In total you can setup everything in at most 30 minutes. The AME can power the station long enough for you to get meme gour engine up and running safely.

**Comment by Eneocho** [comment]
> 
> 
> you only need 5 gold, silver, uranium, and plasma. Then before you are mining order additional metal glass and plastic. That all will take you at most 10 minutes to do. The longest part is literally waiting on the flatpack machine to print you everything. In total you can setup everything in at most 30 minutes. The AME can power the station long enough for you to get meme gour engine up and running safely.

On a recent round I built a gour engine in an hour and a half or around that. I built it in space so I had to also make the entire structure and resize it when I ran outta space. Fair, I am new to gour engine building so I had inefficiencies but assuming I slice them out, it takes at least 50 minutes. It's not that fast as you say, man. And for such a time investment you could've made a R-UST or a shard which are (in my opinion) way more fun engines to design.

**Comment by AutismSpider** [comment]
Just remove silly wall phasing they can do.

**Pull request closed without merging**